metadata.go

```diff
@@ -84,10 +84,27 @@ type AggregateMetadata struct {
 
 // ViewMetadata holds the metadata for views.
 type ViewMetadata struct {
-	Keyspace   string
-	Name       string
-	FieldNames []string
-	FieldTypes []TypeInfo
+	Keyspace                string
+	Name                    string
+	BaseTableId             UUID
```

Comment:
What do you think about having a pointer to the base table? possibly instead of ID and name?

```go
BaseTable *TableMetadata
```

Reply:
I think it's a good idea, but it would only replace BaseTableName right? Because TableMetadata doesn't have the BaseTableId.